tf/examples/Bonsai/bonsai_example.py

```diff
@@ -74,21 +78,28 @@ def main():
 
     helpermethods.dumpCommand(sys.argv, currDir)
 
+    print("Num of classes : ", numClasses)
     # numClasses = 1 for binary case
     bonsaiObj = Bonsai(numClasses, dataDimension,
-                       projectionDimension, depth, sigma)
+                       projectionDimension, depth, sigma, isRegression=isRegression)
```

Comment:
No need of isRegression=isRegression. The variable names are there to be self explanatory